doc/cli/npm-install.md

```diff
@@ -155,71 +155,70 @@ after packing it up into a tarball (b).
           npm install sax@">=0.1.0 <0.2.0"
           npm install @myorg/privatepackage@">=0.1.0 <0.2.0"
 
-* `npm install <githubname>/<githubrepo>`:
+* `npm install <git remote url>`:
 
-    Install the package at `https://github.com/githubname/githubrepo` by
-    attempting to clone it using `git`.
+    Install a package by cloning a git remote url.  The format of the git
+    url is:
 
-    Example:
+          <protocol>://[<user>[:<password>]@]<hostname>[:<port>][:/]<path>[#<commit-ish>]
 
-          npm install mygithubuser/myproject
+    `<protocol>` is one of `git`, `git+ssh`, `git+http`, or
+    `git+https`.  If no `<commit-ish>` is specified, then `master` is
+    used.
 
-   To reference a package in a generic git repo (not on GitHub), see git remote
-   urls below.
+    Examples:
 
-* `npm install github:<githubname>/<githubrepo>`:
+          git+ssh://git@github.com:npm/npm.git#v1.0.27
+          git+https://isaacs@github.com/npm/npm.git
+          git://github.com/npm/npm.git#v1.0.27
 
-    The same as the above, but explicitly marked as a GitHub dependency.
+* `npm install <githubname>/<githubrepo>[#<commit-ish>]`:
+* `npm install github:<githubname>/<githubrepo>[#<commit-ish>]`:
 
-    Example:
+    Install the package at `https://github.com/githubname/githubrepo` by
+    attempting to clone it using `git`.
+
+    If you don't specify a *commit-ish* then `master` will be used.
```

Comment:
Might as well spell it the way we do in our own discussions, and ditch the hyphen (maybe with a brief explanation of what constitutes a commitish for npm purposes?